src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -1158,8 +1158,11 @@ public void run()
         }
         catch (SocketException ex)
         {
-            if (!"Socket closed".equals(ex.getMessage()))
+            if (!"Socket closed".equals(ex.getMessage())
```

Comment:
the string comparison seems like a bit of a bummer.  is it possible other exceptions (which we don't know the types of) might throw that same error message?